### PR TITLE
[Test] testing integration_test

### PR DESCRIPTION
**Pull request by sandy1618**
## Make Integration_test is now working for openAI, Azure and claude.

- edited action files to include integration_test. 
- First tested with PR action file and it working . Added new git secrets for the same. 
- https://github.com/gaudiy/langsmith-evaluation-helper/actions/runs/10192623113/job/28195949566
- Have now moved it to release.yml

### CODE REVIEW

**Comment by sandy1618** [comment]
# Make Integration_test is now working for openAI, Azure and claude.

File: .github/workflows/pr-check.yml [hunk 1]
```diff
@@ -60,4 +60,7 @@ jobs:
           AZURE_OPENAI_API_KEY: ${{ secrets.AZURE_OPENAI_API_KEY }}
           ANTHROPIC_API_KEY: ${{ secrets.ANTHROPIC_API_KEY }}
         run: | 
-          make unit_test
\ No newline at end of file
+          make unit_test
+
```
Comment (thebongcook) [inline .github/workflows/pr-check.yml:64]: Any reason for these blank lines?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -43,6 +43,20 @@ jobs:
           uv pip sync --system requirements.txt
           uv pip install --system -e .
 
+      - name: Run Integration
```
Comment (kazuyaseki) [inline .github/workflows/release.yml:46]:
```suggestion
      - name: Run Integration Test
```

**Review by kazuyaseki — APPROVED** [review]
Looks good, left a minor comment

**Review by thebongcook — APPROVED** [review]
LGTM

File: .github/workflows/pr-check.yml (hunk 1 above)
Comment (sandy1618) [reply .github/workflows/pr-check.yml:64]: Thanks, I have removed the blank line!

**Pull request merged**